src/robot/parsing/lexer/tokens.py

```diff
@@ -49,6 +49,7 @@ class Token:
     KEYWORD_NAME = 'KEYWORD NAME'
 
     DOCUMENTATION = 'DOCUMENTATION'
+    SUITE_NAME = 'SUITE NAME'
```

Comment:
It's a bit unfortunate we apparently need to use `SUITE_NAME` here instead of `NAME` that's already used elsewhere.

Reply:
Probably not the best time to ask, but after going through this PR one more time I feel like I'm confused about which Token should i use... either reuse `Token.NAME` or the new one `Token.SUITE_NAME` :smile: